pkg/geo/wkt/generate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+goyacc -o wkt_generated.go -p "wkt" wkt.y
+#todo (ayang) modify goyacc to accept a cmd line flag to set this
+sed -i '' 's/wktErrorVerbose = false/wktErrorVerbose = true/' wkt_generated.go
```

Comment:
you'll need to make this `sed` linux compatible i think (`sed -i ''` on OSX/BSD, `sed -i` on everything else). may be easier to use `awk`... or write a go wrapper that does the replace and thus environment proof, e.g.

generate_wkt.go
```
// +build do_not_build

package main

func main() {
   // run goyacc
   // replace regex and write file
}
```

and then
```
//go:generate go run generate_wkt.go
```

Reply:
Discussed offline that using `sed` with a temp file should work